docs/developers/Building_Windows.md

```diff
@@ -10,7 +10,10 @@ Windows builds can be made in more than one way and with more than one tool. Thi
 - Git or git GUI, for example, SourceTree [download link](http://www.sourcetreeapp.com/download)
 - CMake [download link](https://cmake.org/download/). During install after accepting license agreement make sure to check "Add CMake to the system PATH for all users".
 - To unpack pre-build Vcpkg: [7-zip](http://www.7-zip.org/download.html)
-- Optionally, to create installer: [NSIS](http://nsis.sourceforge.net/Main_Page)
+- To speed up recompilation: [CCache](https://github.com/ccache/ccache/releases)
```

Comment:
That's also optional

Reply:
I'll need to find out how to use CMake variables when building with Visual Studio